apps/epic-react/src/components/app/navigation/index.tsx

```diff
@@ -0,0 +1,342 @@
+              cx(
+                'rounded-md px-3 py-2 text-text transition-opacity duration-150 ease-in-out hover:opacity-100 sm:opacity-75',
+                {'bg-er-gray-100 sm:opacity-100': location === '/articles'},
+              ),
+            )}
+          >
+            Articles
+          </Link>
+          <Link
+            href="/livestreams"
+            className={twMerge(
+              cx(
+                'rounded-md px-3 py-2 text-text transition-opacity duration-150 ease-in-out hover:opacity-100 sm:opacity-75',
+                {'bg-er-gray-100 sm:opacity-100': location === '/livestreams'},
+              ),
+            )}
+          >
+            Livestreams
+          </Link>
+          <Link
+            href="/podcast/kents-career-path-through-web-development"
+            className={twMerge(
+              cx(
+                'rounded-md px-3 py-2 text-text transition-opacity duration-150 ease-in-out hover:opacity-100 sm:opacity-75',
+                {
+                  'bg-er-gray-100 sm:opacity-100':
+                    location.startsWith('/podcast/'),
+                },
+              ),
+            )}
+          >
+            Podcast
+          </Link>
+          <Link
+            href="/faq"
+            className={twMerge(
+              cx(
+                'rounded-md px-3 py-2 text-text transition-opacity duration-150 ease-in-out hover:opacity-100 sm:opacity-75',
+                {'bg-er-gray-100 sm:opacity-100': location === '/faq'},
+              ),
+            )}
+          >
+            FAQ
+          </Link>
+          {/* TODO: sellingLive? */}
+          {sellingLive && !isAuthenticated ? (
+            <Link
+              href="/login"
+              className="rounded-lg px-3 py-2 text-text transition-all duration-150 ease-in-out lg:bg-blue-500 lg:text-white lg:hover:bg-blue-600"
+            >
+              Restore Purchases
+            </Link>
+          ) : null}
+
+          {isAuthenticated &&
+          !isEmpty(sessionData?.user?.purchases) &&
+          !purchasedOnlyTeam ? (
+            <Link
+              href="/learn"
+              className={twMerge(
+                cx(
+                  'rounded-lg border-transparent px-3 py-2 leading-tight text-text transition-colors duration-150 ease-in-out  md:border md:bg-blue-500 md:text-white md:hover:bg-blue-600',
+                  {
+                    'bg-background text-text sm:opacity-100 md:bg-background md:text-text md:hover:text-white':
+                      location === '/learn',
+                  },
+                ),
+              )}
+            >
+              Workshops
+            </Link>
+          ) : null}
+          <div
+            className={twMerge(
+              cx('flex items-start', {
+                'flex-col': isTablet,
+                'flex-row items-center': !isTablet,
+              }),
+            )}
+          >
+            {/* TODO: <Feedback /> component */}
```

Comment:
Feedback component is not implemented yet - have to investigate how it connect to the API